types/es-abstract/5/FromPropertyDescriptor.d.ts

```diff
@@ -0,0 +1,4 @@
+import type { PropertyDescriptor } from '../index';
+
+declare function FromPropertyDescriptor<T = unknown>(Desc: PropertyDescriptor<T>): TypedPropertyDescriptor<T>;
```

Comment:
can you elaborate on what's happening here?  is `TypedPropertyDescriptor` a builtin thing?

Reply:
`TypedPropertyDescriptor` is a **TypeScript** built‑in interface from `lib.es5.d.ts`.